src/cron-schedule.ts

```diff
@@ -0,0 +1,206 @@
+/**
+ * Represents a cron schedule
+ */
+export abstract class CronSchedule {
+
+  /**
+   * Create a cron schedule which runs every minute
+   */
+  public static everyMinute(): CronSchedule {
+    const minute = '*';
+    const hour = '*';
+    const day = '*';
+    const month = '*';
+    const weekDay = '?';
+
+    return new class extends CronSchedule {
+      public readonly expressionString: string = `${minute} ${hour} ${day} ${month} ${weekDay}`;
+      public _bind() {
+        return new LiteralSchedule(this.expressionString);
+      }
+    };
```

Comment:
All of this can be replaced with:

```suggestion
  return CronSchedule.custom({ minute: '*', hour: '*', day: '*', month: '*', weekday: '?' })
```

Also, looking at [CrontabGuru](https://crontab.guru/#*_*_*_*_*), every minute evaluates to `* * * * *`. Where does the `?` come from? Is it valid in Cron?

Reply:
So I was following [Quartz Cron Expression format](http://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/crontrigger.html) which I believe is being followed in CDK. 
So in this format, day and weekday cannot be specified at the same time, one of them needs to be `?`. 
It is working locally in the cluster but I see [CronJob Kubernetes documentation](http://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/crontrigger.html) mentioning [Wikipedia](https://en.wikipedia.org/wiki/Cron) link as cron format. If I follow that, then CronTabGuru would be the format.

Reply:
I see K8s [documentation](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#cron-schedule-syntax) mention CronTab Guru so probably best to follow that. Will make the updates. Good catch 👍🏻